docs/nb/Debuggin_H0.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        if pval &lt; 64.4:
</pre><p>is this supposed to be pval &gt; 64.4 pval &lt; 65?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/FRBs/zdm/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>